deployment/kustomizations/helm-values-qdrant_0.8.5.yaml

```diff
@@ -0,0 +1,234 @@
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+args: ["./config/initialize.sh"]
+env: {}
+  # - name: QDRANT_ALLOW_RECOVERY_MODE
+  #   value: true
+
+# checks - Readiness and liveness checks can only be enabled for either http (REST) or grpc (multiple checks not supported)
+# grpc checks are only available from k8s 1.24+ so as of per default we check http
+service:
+  type: ClusterIP
+  additionalLabels: {}
+  annotations: {}
+  loadBalancerIP: ""
+  ports:
+    - name: http
+      port: 6333
+      targetPort: 6333
+      protocol: TCP
+      checksEnabled: true
+    - name: grpc
+      port: 6334
+      targetPort: 6334
+      protocol: TCP
+      checksEnabled: false
+    - name: p2p
+      port: 6335
+      targetPort: 6335
+      protocol: TCP
+      checksEnabled: false
+
+ingress:
+  enabled: false
+  ingressClassName: "nginx"
+  additionalLabels: {}
+  annotations:
+    # Dependancy: https://kubernetes-sigs.github.io/external-dns
+    #external-dns.alpha.kubernetes.io/filter: 'include'
+    #external-dns.alpha.kubernetes.io/cloudflare-proxied: 'true'
+    external-dns.alpha.kubernetes.io/target: ingress1.myhost.net
+  hosts:
+    - host: qdrant-ai.myhost.net
+      paths:
+        - path: /
+          pathType: Prefix
+          servicePort: 6333
+  tls:
+    - hosts:
+       - qdrant-ai.myhost.net
+      secretName: qdrant-ai.myhost.net-tls
+
+livenessProbe:
+  enabled: false
+  initialDelaySeconds: 5
+  periodSeconds: 5
+  timeoutSeconds: 1
+  failureThreshold: 6
+  successThreshold: 1
+
+readinessProbe:
+  enabled: true
+  initialDelaySeconds: 5
+  periodSeconds: 5
+  timeoutSeconds: 1
+  failureThreshold: 6
+  successThreshold: 1
+
+startupProbe:
+  enabled: false
+  initialDelaySeconds: 10
+  periodSeconds: 5
+  timeoutSeconds: 1
+  failureThreshold: 30
+  successThreshold: 1
+
+additionalLabels: {}
+podAnnotations: {}
+podLabels: {}
+
+resources: {}
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+containerSecurityContext:
+  runAsNonRoot: true
+  runAsUser: 1000
+  runAsGroup: 2000
+  allowPrivilegeEscalation: false
+  privileged: false
+  readOnlyRootFilesystem: true
+
+podSecurityContext:
+  fsGroup: 3000
+  fsGroupChangePolicy: Always
+
+lifecycle:
+  preStop:
+    exec:
+      # Sleeping before shutdown allows Qdrant to process requests that were
+      # in-flight before the node is removed from load-balancing.
+      # If using an external load balancer, you may need to increase this
+      # duration to be greater than the LB's health check interval.
+      command: ["sleep", "3"]
+
+# If true ensures that the pre-existing files on the storage and snapshot volume are owned by the container's
+# user and fsGroup
+updateVolumeFsOwnership: true
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+topologySpreadConstraints: []
+
+persistence:
+  accessModes: ["ReadWriteOnce"]
+  size: 20Gi
+  annotations: {}
+#  storageClassName: vsphere-retain
```

Comment:
I think you kept this comment becuase you test with vsphere ? Let's use comments qdrant use by default, so users won't get confused ?

```
# storageVolumeName: qdrant-storage
# storageClassName: local-path
```

Reply:
I think we should leave this as is or remove comment `#  storageClassName: vsphere-retain`.
If no storageClassName, the default storage class will be used. @wwwy3y3

Reply:
I see. Then I think we can leave this as is. Would be a nice reference.